tests/Api/Admin/ProductTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Component\Core\Model\ProductImageInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ProductTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_returns_product_collection(): void
+    {
+        $this->loadFixturesFromFiles(['product/product_variant_with_original_price.yaml', 'authentication/api_administrator.yaml']);
+
+        $this->client->request(
+            'POST',
+            '/api/v2/admin/authentication-token',
+            [],
+            [],
+            ['CONTENT_TYPE' => 'application/json', 'HTTP_ACCEPT' => 'application/json'],
+            json_encode(['email' => 'api@example.com', 'password' => 'sylius'])
+        );
+
+        $token = json_decode($this->client->getResponse()->getContent(), true)['token'];
+        $authorizationHeader = self::$container->getParameter('sylius.api.authorization_header');
+
+        $header['HTTP_' . $authorizationHeader] = 'Bearer ' . $token;
```

Comment:
if I see correctly you use it twice here and in the next scenario...
but what is more important why didn't you use `tests/Api/Utils/AdminUserLoginTrait.php`?